<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tab_title">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab_options">
      <attribute name="title">
       <string>Options</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="vspacer_options">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_models">
      <attribute name="title">
       <string>Models</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="grid_models">
         <property name="verticalSpacing">
          <number>18</number>
         </property>
         <item row="3" column="0">
          <widget class="QLabel" name="label_units_list">
           <property name="text">
            <string>Units List:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_models_database">
           <property name="text">
            <string>Models Database</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <layout class="QVBoxLayout" name="vlayout_models_database_actions">
           <item>
            <widget class="QPushButton" name="push_models_database_new">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>New</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="push_models_database_save">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="push_models_database_revert">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Revert</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_search">
           <property name="text">
            <string>Search:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <layout class="QVBoxLayout" name="vlayout_models_buttons">
           <item>
            <widget class="QPushButton" name="push_models_create">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Create Model</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="push_edit_model">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Edit Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="push_models_delete">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Delete Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="vspacer_models_actions">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="2">
          <widget class="QComboBox" name="combo_search_mode">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <item>
            <property name="text">
             <string>And</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Or</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="edit_models_search"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_models_list">
           <property name="text">
            <string>Models List:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="combo_models_database">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <layout class="QVBoxLayout" name="vlayout_units_actions">
           <item>
            <widget class="QPushButton" name="push_units_create">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Create Unit</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="push_units_edit">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Edit Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="push_units_delete">
             <property name="focusPolicy">
              <enum>Qt::NoFocus</enum>
             </property>
             <property name="text">
              <string>Delete Selected</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="vspacer_units_actions">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeType">
              <enum>QSizePolicy::MinimumExpanding</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>1</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="2" column="1">
          <widget class="QListWidget" name="list_models">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QListWidget" name="list_units">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::MultiSelection</enum>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="vspacer_models">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="hlayout_options_buttons">
     <item>
      <widget class="QPushButton" name="push_exit_match">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Start Ttitle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="push_exit_title">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Exit Ttitle</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>push_exit_match</tabstop>
  <tabstop>push_exit_title</tabstop>
  <tabstop>combo_models_database</tabstop>
  <tabstop>push_models_database_new</tabstop>
  <tabstop>push_models_database_save</tabstop>
  <tabstop>push_models_database_revert</tabstop>
  <tabstop>edit_models_search</tabstop>
  <tabstop>combo_search_mode</tabstop>
  <tabstop>push_models_create</tabstop>
  <tabstop>push_edit_model</tabstop>
  <tabstop>push_models_delete</tabstop>
  <tabstop>push_units_create</tabstop>
  <tabstop>push_units_edit</tabstop>
  <tabstop>push_units_delete</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
